src/js/components/BaseWidget.js

```diff
@@ -17,7 +17,7 @@ class BaseWidget {
 
     const newValue = thisWidget.parseValue(value);
 
-    if (thisWidget.correctValue !== newValue && thisWidget.isValid(newValue)) {
+    if (newValue !== thisWidget.correctValue && thisWidget.isValid(newValue)) {
```

Comment:
Nie bardzo rozumiem po co ta zmiana, bo nic nie wnosi. Sprawdzasz dokładnie to samo ;)